CHANGES.md

```diff
@@ -3,6 +3,7 @@ Change Log
 
 ### 1.40 - 2017-12-01
 
+* Adds Material support to Globe. [#5919](https://github.com/AnalyticalGraphicsInc/cesium/pull/5919)
```

Comment:
Can this link to the future Sandcastle example that will live here: https://cesiumjs.org/Cesium/Apps/Sandcastle/